src/validateEmail.test.js

```diff
@@ -17,4 +19,29 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   // write more tests here
+  it(`should return 'false' for the invalid email`, () => {
+    expect(validateEmail('')).toBeFalsy();
+  });
+
+  it(`should return 'false' for the invalid email that
+  contains not allowed chars
+  ! $ % & ' * + / = ? ^ { | } ~ :
```

Comment:
The test description mentions several special characters that are not allowed in an email address. Ensure that the `validateEmail` function correctly handles these characters.